web/pandas/community/ecosystem.md

```diff
@@ -137,7 +137,7 @@ attributes like DataFrame columns.
 
 ### [Jupyter Notebook / Jupyter Lab](https://jupyter.org)
 
-Jupyter Notebook is a web application for creating Jupyter notebooks. A
+Jupyter Lab is a web application for creating Jupyter notebooks. A
```

Comment:
It actually should be `JupyterLab` (no space).

If you could fix the header as well (just two lines above), that would be great.